### PR TITLE
persist golang hash in tuxbot

### PR DESCRIPTION
**Pull request by heronhaye**

### CODE REVIEW

File: packaging/linux/docker_build.sh [hunk 1]
```diff
@@ -59,7 +59,7 @@ gpg_tempfile="$gpg_tempdir/code_signing_key"
 gpg --export-secret-key --armor "$code_signing_fingerprint" > "$gpg_tempfile"
 
 # Make sure the Docker image is built.
-image=keybase_packaging_v19
+image=keybase_packaging_v21
```
Comment (heronhaye) [inline packaging/linux/docker_build.sh]: bumped twice because i was messing around with v20 on tuxbot

File: packaging/linux/Dockerfile [hunk 2]
```diff
@@ -28,7 +28,14 @@ RUN apt-get install -y nodejs yarn
 
 # Install Go directly from Google, because the Debian repos tend to be behind.
 # When updating the version, remember to bump keybase_packaging_v# in docker_build.sh.
-RUN wget "https://storage.googleapis.com/golang/go1.12.4.linux-amd64.tar.gz" -O /root/go.tar
-RUN tar -C /usr/local -xzf /root/go.tar
-RUN rm /root/go.tar
+# Copy over the new hash when upgrading version. But if not upgrading version,
+# the check should not ever fail.
+ENV GOLANG_VERSION 1.12.4
+ENV GOLANG_DOWNLOAD_URL https://golang.org/dl/go$GOLANG_VERSION.linux-amd64.tar.gz
```
Comment (heronhaye) [inline packaging/linux/Dockerfile]: note url change

File: packaging/linux/Dockerfile [hunk 3]
```diff
@@ -28,7 +28,14 @@ RUN apt-get install -y nodejs yarn
 
 # Install Go directly from Google, because the Debian repos tend to be behind.
 # When updating the version, remember to bump keybase_packaging_v# in docker_build.sh.
-RUN wget "https://storage.googleapis.com/golang/go1.12.4.linux-amd64.tar.gz" -O /root/go.tar
-RUN tar -C /usr/local -xzf /root/go.tar
-RUN rm /root/go.tar
+# Copy over the new hash when upgrading version. But if not upgrading version,
+# the check should not ever fail.
+ENV GOLANG_VERSION 1.12.4
```
Comment (joshblum) [inline packaging/linux/Dockerfile]: should we bump to 1.12.5 if we need that for mobile anyway?

Comment (heronhaye) [reply packaging/linux/Dockerfile]: Thought we needed something not released yet for mobile?

**Pull request merged**